src/main/java/li/cil/scannable/data/ModItemModelProvider.java

```diff
@@ -3,22 +3,21 @@
 import li.cil.scannable.api.API;
 import li.cil.scannable.common.item.Items;
 import net.minecraft.data.DataGenerator;
+import net.minecraft.data.models.ModelProvider;
 import net.minecraft.resources.ResourceLocation;
 import net.minecraft.world.item.Item;
-import net.minecraftforge.client.model.generators.ItemModelBuilder;
-import net.minecraftforge.client.model.generators.ItemModelProvider;
-import net.minecraftforge.common.data.ExistingFileHelper;
 
 import java.util.Objects;
 
-public final class ModItemModelProvider extends ItemModelProvider {
-    public ModItemModelProvider(final DataGenerator generator, final ExistingFileHelper existingFileHelper) {
-        super(generator, API.MOD_ID, existingFileHelper);
+/*
```

Comment:
WIP/fell between the cracks/unneeded?

Reply:
Model generation wasn't supported in Fabric at the time that I ported to 1.18.1. I think it is supported in 1.18.2 so I'll look at adding it back.

Reply:
I've added this back now.